### ISSUE TITLE
Unable to place floor tiles

### ISSUE THREAD

**Issue by sircules** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#408]
**Describe the bug**
When placing floor tile instead of the typical ‘2x2’ format and placing the tile, it will instead show large negative number by large negative number and not allow tiles to be placed.

This happens every now and then while placing tiles and then resolves itself after a couple tries. 

**Additional context**
This bug is not majorly impacting as it does go away but I just want to let you guys know.  Keep up the good work!

**Comment by LuPro** [comment]
Could it be that you tried moving (accidentally moved) the build plane at the same time like in #266? If you're not changing the build plane at the same time, is it still showing similar numbers? Could be the same error, but caused by two different interactions then.

**Comment by sircules** [comment]
Yes. It’s similar looks similar to the photo on #266. I don’t know specifically if it was moving the build plane (as you said accidentally). I just know on occasion that when filling certain areas it has happened. It hasn’t happened since last night in any of the floors I’ve built since. If you think it’s related to the same issue as #266 then you’re probably right, sorry for the added work load.

**Comment by LuPro** [comment]
No problem :) I always rather have someone adding a "maybe duplicate" than not adding the issue at all - I'll keep it open in case it is from another source though I suspect it's the same - once we have a fix, we can check if both issues don't come up anymore
